src/gui/mainwindow.cpp

```diff
@@ -1887,15 +1887,17 @@ void MainWindow::on_actionSearchWidget_triggered()
 
 #ifdef Q_OS_WIN
             const QMessageBox::StandardButton buttonPressed = QMessageBox::question(this, tr("Old Python Runtime")
-                , tr("Your Python version (%1) is outdated. Minimum requirement: 3.5.0.\nDo you want to install a newer version now?")
+                , tr("Your Python version (%1) is outdated. Minimum requirement: %2.\nDo you want to install a newer version now?")
                     .arg(pyInfo.version)
+                    .arg("3.5.0")
```

Comment:
```suggestion
                    .arg(pyInfo.version, QLatin1String("3.5.0"))
```

Reply:
will change, are you ok with commit title `Make Minimum requirement an argument`?

Reply:
Perhaps: `Make Python minimum version requirement an argument`